tests/conftest.py

```diff
@@ -57,12 +56,12 @@ def celery_config():
 
 @pytest.fixture(scope="session")
 def job_a() -> Task[P, str]:
-    @task(name="job_a")
-    def job(*args: P.args, **kwargs: P.kwargs) -> str:
+    @subscribe_to(topic="index.high")
```

Comment:
Let's not modify existing tests here. Let's add new ones instead.

And, we'd need a test with
```python
@subscribe_to(...)
def x(...): ...
```

and also one with 
```python
@subscribe_to(...)
@task(bind=True)  # or anything else specific, to check that we didn't lose that
def y(self, ...): ...
```